app/[locale]/reservations/[roomType]/[roomName]/helper/calendar/NavigateButtons.tsx

```diff
@@ -111,3 +117,26 @@ const useDateQuery = () => {
     router.push(`${pathname}?selectedDate=${str}`, { scroll: false });
   };
 };
+
+const useClickOutside = <T extends HTMLElement>(handler: () => void) => {
+  const ref = useRef<T | null>(null);
+
+  useEffect(() => {
+    const handleClickOutside = (ev: MouseEvent | TouchEvent) => {
+      const target = ev.target as Node;
```

Comment:
```suggestion
      const target = ev.target;
      if (!(ev.target instanceof Node)) return;
```

(프로젝트와는 관련없는 사람이지만) `as` 보다는 타입 가드를 쓰면 좋을 것 같습니당!

Reply:
반영했습니다! 감사해요~